src/relay/backend/interpreter.cc

```diff
@@ -209,6 +242,25 @@ class Interpreter :
 
   Value InvokePrimitiveOp(Function func,
                           const Array<Value>& args) {
+    auto call_node = func->body.as<CallNode>();
+    if (call_node) {
+      auto debug = Op::Get("debug");
+      if (call_node->op == debug) {
+        auto dattrs = call_node->attrs.as<DebugAttrs>();
+        auto is = this->get_state(call_node->args[0]);
+        if (dattrs->debug_func.defined()) {
+          dattrs->debug_func(is);
+        } else {
+          RELAY_DEBUG(is);
+        }
+        auto kont = FunctionNode::make(
```

Comment:
I am confused by this continuation. shouldnt we evaluate the value instead as it is in invokePrimitive, with outer args supplied? if my assumption is correct the test will return const 1 instead of 1, can you check?

Reply:
This was right in spirit, wrong in implementation. We can just return the first value.